packages/typegpu/src/index.ts

```diff
@@ -17,7 +18,7 @@ export const tgpu = {
 
   bindGroupLayout,
 
-  init,
+  init: init as (options?: InitOptions) => Promise<TgpuRoot>,
```

Comment:
Why does it have to be cast? Isn't the type already that?

Reply:
Not on the 0.2 branch, it still returns ExperimentalTgpuRoot there